shotover-proxy/src/message/mod.rs

```diff
@@ -434,27 +483,61 @@ impl Value {
                 IpAddr::V4(four) => Vec::from(four.octets()),
                 IpAddr::V6(six) => Vec::from(six.octets()),
             }),
-            _ => unimplemented!(),
+            Value::List(_) => todo!(),
```

Comment:
Lets delete into_bytes instead.
Its unused and I have no idea what its intended use is.
In the current implementation there is no way to revert the data back as there is no tag to identify what the original type was.